run_dir/static/js/project_samples.js

```diff
@@ -1232,3 +1233,26 @@ function old_project_warning(warndate_raw){
     $('#old_project_warning').attr('title', 'This project was created before '+warndate_raw+'.<br>Genomics Status may be inaccruate.');
   }
 }
+function load_charon_summary(){
+  $.getJSON("/api/v1/charon_summary/"+ project, function(data) {
+      if (data['tot']==0){
+          $('#tab_charon_data').html("This project has no data in Charon.");
```

Comment:
No because people might be browsing the source code directly, and then we will tell them that there is no data in charon !